nix/searxng.nix

```diff
@@ -0,0 +1,94 @@
+{ pkgs
+, lib
+, name
+, config
+, ...
+}:
```

Comment:
These should be in a single line, to be consistent with the formatting of the rest of the files.

Reply:
I used the default `treefmt`, how about letting the tool format the file and avoid these discussions (IMHO, it's a waste of time) ?
If there's something wrong with the formatting, let's just fix the underlying tool and fix all the files at once in a follow up PR.

Reply:
`nixpkgs-fmt`, which `treefmt` uses, **does not** *reformat* one-line arguments into multi-line arguments. The point here is that formatting should be _consistent_ with the rest of the files. You should not introduce inconsistent formatting.

Reply:
OK fine.
I just pushed the changes.